frontend/src/helper.js

```diff
@@ -143,3 +143,7 @@ export const shortenedDisplayName = displayName => {
 export function makePermissionReadable(intent) {
   return strings.permissions[intent.replace(/[.]/g, "_")] || intent;
 }
+
+export const dateFormat = () => {
```

Comment:
If the function doesn't do anything but giving a constant, it should be a constant.
### Suggestion:
const getDate = (time) => {
dayjs(time).format("MM.DD.YYYY")
}